doc/docs.md

```diff
@@ -4040,6 +4040,13 @@ println(compare(1.1, 1.2)) //         -1
 V's model of concurrency is going to be very similar to Go's.
 For now, `spawn foo()` runs `foo()` concurrently in a different thread:
 
+> **Note**
+> Threads relies on running machine's CPU (number of cores/threads).
+> Be aware that OS threads spawned with `spawn`
+> have limitations in regard to concurrency, 
+> including resource overhead and scalability issues, 
+> and might affect performance in cases of high thread count.
+
```

Comment:
imho this note should be after the example, and before the `go` paragraph, i.e. in line 4069 . Putting it here, breaks the sentence from the example that follows it.